packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -451,7 +467,8 @@ macro_rules! impl_tx_wrapper {
                         _ => {
                             // not a contract, it's safe to unwrap
                             let owner = extract_owner_or_recipient(input).unwrap();
-                            let asset_id = input.asset_id().unwrap().to_owned();
+                            let asset_id = input.asset_id(&BASE_ASSET_ID).unwrap().to_owned();
```

Comment:
I guess we missed this `unwrap` here and on the line above. Maybe a good opportunity to properly handle these two errors?

Reply:
they are safe to unwrap as we exclude `Contract`